server/routerlicious/packages/lambdas/src/deli/lambdaFactory.ts

```diff
@@ -198,17 +209,19 @@ export class DeliLambdaFactory extends EventEmitter implements IPartitionLambdaF
                         "lastAccessTime": Date.now(),
                     };
                     await this.collection.update(query, data, null);
-                    context.log?.info(
-                        `Marked session alive and active as false for closeType: ${JSON.stringify(closeType)}`,
-                        { messageMetaData },
-                    );
+                    const message = `Marked session alive and active as false for closeType:
+                        ${JSON.stringify(closeType)}`;
+                    context.log?.info(message, { messageMetaData });
+                    Lumberjack.info(message, { messageMetaData });
                 }
             };
             handler().catch((e) => {
+                const message = `Failed to handle session alive and active with exception ${e}`;
                 context.log?.error(
-                    `Failed to handle session alive and active with exception ${e}`,
+                    message,
                     { messageMetaData },
                 );
+                Lumberjack.error(message, { messageMetaData });
```

Comment:
Let's extract the error `e` and log it like:

`Lumberjack.error(message, getLumberBaseProperties(documentId, tenantId), e);`